horace_core/@d3d_old/cut.m

```diff
@@ -52,14 +52,21 @@
 
 % Perform operations
 % ------------------
+if iscell(w.data)&& ischar(w.data{1})
+    % Slightly hacky - load dnd object before passing it to sqw.cut so we don't
+    % attempt to load the file in as an SQW file. We reach this point if cut_dnd
+    % is called.
+    % After DnDBase class is implemented, cutting a DnD object need no longer
+    % go through sqw.cut and this file will be largely re-written
+    w.data = d3d(w.data{1});
```

Comment:
`d3d_old`? (same for the other repetitions of this code fragment)

Reply:
@nickbattam-tessella This was automatically merged from master when it was d3d. I can fix this (and similar) back to `_old` , thanks for spotting